auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+        except OTPVerification.DoesNotExist:
+            logger.warning(
+                f"Session not found for revocation: "
+                f"user={user.username}, session={session_id}"
+            )
+            return False
+        except Exception as e:
+            logger.error(f"Error revoking session: {e}")
+            return False
+    
+    @staticmethod
+    def cleanup_expired_blacklist():
```

Ruff (0.12.2)

Missing return type annotation for staticmethod `cleanup_expired_blacklist`

(ANN205)